factorial.sh

```diff
@@ -2,14 +2,22 @@
 
 factorial(){
     local num=$1
-    local a=1
-    while (($num > 1))
-        do
-        a=$(($a * $num))
-        ((num=$num-1))
-        done
-    echo $a
+    if (($num > 1)); then
+        local a=1
+        while (($num > 1))
```

Comment:
If you change this to `$num >= 1` does that mean you  don't need the extra check below?